packages/embarkjs/build-test/blockchain.js.map

```diff
@@ -0,0 +1 @@
WAVD;AAWD,SAZK,CAAN,CAb0B,CA2B1B;;AACA,SAAC;AAACnB,UAAAA,OAAD;AAAUW,UAAAA;AAAV,YAA4B,MAAML,WAAW,EAA9C;;AACA,YAAI;AACF,gBAAMd,UAAU,CAACuB,OAAX,CAAmB;AAACJ,YAAAA;AAAD,WAAnB,CAAN;AACD,SAFD,CAEE,OAAOQ,CAAP,EAAU;AACV,cAAI,CAACf,QAAQ,CAACF,KAAd,EAAqB,MAAMiB,CAAN;AACtB;;AACDnB,QAAAA,OAAO,CAACG,OAAR,CAAgB,CAACM,MAAD,EAASQ,GAAT,KAAiB;AAC/B1B,UAAAA,MAAM,CAAC2B,WAAP,CAAmBT,MAAM,CAACR,OAA1B,EAAmCG,QAAQ,CAACH,OAAT,CAAiBgB,GAAjB,CAAnC;AACD,SAFD;AAGD,OArCC,CAAF;AAsCD,KAvCD;AAwCD,GAnEO,CAAR;AAoED,CArEO,CAAR","sourcesContent":["/* global before describe it require */\n\nconst {startRPCMockServer, TestProvider} = require('./helpers/blockchain');\nconst {assert} = require('chai');\nconst {Blockchain} = require('..');\nconst {promisify} = require('util');\n\ndescribe('Blockchain', () => {\n  describe('#connect', () => {\n    before(() => {\n      Blockchain.registerProvider('web3', TestProvider);\n      Blockchain.setProvider('web3', {});\n    });\n\n    const scenarios = [\n      {\n        description: 'should not keep trying other connections if connected',\n        servers: [true, true],\n        visited: [true, false],\n        error: false\n      },\n      {\n        description: 'should keep trying other connections if not connected',\n        servers: [false, true],\n        visited: [true, true],\n        error: false\n      },\n      {\n        description: 'should return error if no connections succeed',\n        servers: [false, false],\n        visited: [true, true],\n        error: true\n      }\n    ];\n\n    scenarios.forEach(({description, ...scenario}) => {\n      it(description, async () => {\n        const makeServers = async () => {\n          const servers = await Promise.all(\n            scenario.servers.map(server => (\n              promisify(startRPCMockServer)({successful: server})\n            ))\n          );\n          const dappConnection = servers.map(server => server.connectionString);\n          return {servers, dappConnection};\n        };\n\n        // test Blockchain.connect() using callback\n        let {servers, dappConnection} = await makeServers();\n        await new Promise((resolve, reject) => {\n          Blockchain.connect({dappConnection}, err => {\n            try {\n              assert(scenario.error ? err : !err);\n              servers.forEach((server, idx) => {\n                assert.strictEqual(server.visited, scenario.visited[idx]);\n              });\n              resolve();\n            } catch (e) {\n              reject(e);\n            }\n          });\n        });\n\n        // test Blockchain.connect() without callback\n        ({servers, dappConnection} = await makeServers());\n        try {\n          await Blockchain.connect({dappConnection});\n        } catch (e) {\n          if (!scenario.error) throw e;\n        }\n        servers.forEach((server, idx) => {\n          assert.strictEqual(server.visited, scenario.visited[idx]);\n        });\n      });\n    });\n  });\n});\n"],"file":"blockchain.js"}
```

Comment:
You have some `.map`  files that snucked their way in

Reply:
Fixed, thanks!